### PR TITLE
Updated wallet fix

### PR DESCRIPTION
**Pull request by sneakzttv**
It didn't like the sorting algorithm as the root for json serialization has changed, removing the algorithm now allows wallets to connect. This was only being used to display wallet count anyway and will still error correctly on a null fetch.

There's probably a better way to sort through the count if needed but as this is critical i'd like to get our devs in the builder program using main back to a functional state.

image

image

To Test:
Check that scanning with different wallets works i.e metamask, trust.

Expected Result:
Wallet logins work.

### CODE REVIEW

File: Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/WalletConnectLoginProvider.cs [hunk 1]
```diff
@@ -242,19 +242,9 @@ private IEnumerator FetchSupportedWallets()
 
                 yield return null;
             }
-
             else
             {
-                var json = webRequest.downloadHandler.text;
```
[the remark this answers is not in the document]
Comment (robGG1997) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/WalletConnectLoginProvider.cs:248]: It's not just used to log count it's used to list wallets for redirecting, please check all references/usages before making such decisions

[the remark this answers is not in the document]
Comment (robGG1997) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/WalletConnectLoginProvider.cs:248]: You can find usages in platform dependent compilations using #if directives for IOS and Standalone platforms

[the remark this answers is not in the document]
Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/WalletConnectLoginProvider.cs:248]: no worries, see if you can alter the sorting algorithm if it's needed for supported wallets, then we can keep it in.

[the remark this answers is not in the document]
Comment (kantagara) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/WalletConnectLoginProvider.cs:248]: It's not the sorting algorithm, it's the fact that JSON looks like this 
image
The solution is just to dive into listings and that would resolve our issue.

[the remark this answers is not in the document]
Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/WalletConnectLoginProvider.cs:248]: much cleaner

**Review by kantagara — APPROVED** [review]

**Review by robGG1997 — APPROVED** [review]

**Pull request merged**